test/built-ins/Set/prototype/values/iteration-mutable.js

```diff
@@ -0,0 +1,47 @@
+// Copyright (C) 2014 the V8 project authors. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+  description: >
+      When an item is added to the set after the iterator is created but before
+      the iterator is "done" (as defined by 23.2.5.2.1), the new item should be
+      accessible via iteration. When an item is added to the array after the
+      iterator is is "done", the new item should not be accessible via
```

Comment:
Yes, I repeated this typo a number of times. [The changeset](https://github.com/bocoup/test262/commit/13fe84559330e6b4b09c3c6f94c39cb5ed8255f0) is a little noisy because the correction required re-wrapping the text in some cases.